pages/common/picocom.md

```diff
@@ -0,0 +1,12 @@
+# picocom
+
+> Minimal serial consoles.
+> More information: <https://github.com/npat-efault/picocom>.
```

Comment:
```suggestion
> More information: <https://manned.org/picocom.1>.
```
In general, it is preferred to point to the manual page over pointing to a homepage or something like that. What do you think of changing the link to the manual page on manned.org?

Reply:
I like it!